### ISSUE TITLE
Testing: 2024.3

### ISSUE THREAD

**Issue by ajstrongdev** [rhino-linux/os#92]
**Pre-requesites:**
- [x] Most/all major features be merged
- [x] Tag "core" pacscripts as RC + build

Test 2024.3 RC disk images

Generic:
- [ ] x86_64 
- [ ] ARM64 

Pine64:
- [ ] Pinephone (Pro)
- [ ] Pinetab

Raspberry Pi:
- [ ] Raspberry Pi Desktop
- [ ] Raspberry Pi server

**Individual testers:**
- [ ] ajstrong (generic-x86)
- [ ] oklopfer (generic-arm64 + pine + rpi)
- [ ] elsie19 (generic-x86)
- [ ] DBrox (generic-x86)
- [x] CatMaster21 (generic-x86)

**Comment by ajstrongdev** [comment]
Currently testing x86_64 now in a virtual machine, will update here once I have the results.
I will also be testing on bare metal afterwards, providing all seems fine in VM.

**Comment by cat-master21** [comment]
No issues in a VM also. There are 2 network manager icons/applets that do exactly the same thing, not to big of an issue however.

**Comment by ajstrongdev** [comment]
> No issues in a VM also. There are 2 network manager icons/applets that do exactly the same thing, not to big of an issue however.

Yeah we've had that since release, it's not an issue if it continues, however we should search for a fix soon tbh. I have also finished my testing on x86_64 in a VM and found no issues, will test on bare metal tomorrow and give results.
